modules/govuk_jenkins/templates/jobs/publishing_api_check_validity.yaml.erb

```diff
@@ -0,0 +1,16 @@
+---
+- job:
+    name: Publishing_API_Check_Validity
+    display-name: Publishing_API_Check_Validity
+    project-type: freestyle
+    description: "This job checks the data within the Publishing API for any invalid data"
+    logrotate:
+      artifactNumToKeep: 10
+    publishers:
```

Comment:
I think this will need to be used as [`trigger-parameterized-builds`](http://docs.openstack.org/infra/jenkins-job-builder/publishers.html#publishers.trigger-parameterized-builds)